### PR TITLE
Drop old Symfony and PHP code

### PR DESCRIPTION
**Pull request by jordisala1991**
I am targeting this branch, because this is BC.

### Subject

Removing old code that never gets executed because we dropped SF 2.3 to 2.7

### CODE REVIEW

File: tests/Validator/FormatValidatorTest.php [hunk 1]
```diff
@@ -81,14 +68,7 @@ public function testValidateOnlyReferenceIsAllowedIfNotFormats()
...
 
-        // Prefer the Symfony 2.5+ API if available
-        if (class_exists('Symfony\Component\Validator\Context\ExecutionContext')) {
-            $contextClass = 'Symfony\Component\Validator\Context\ExecutionContext';
-        } else {
-            $contextClass = 'Symfony\Component\Validator\ExecutionContext';
-        }
-
-        $context = $this->getMockBuilder($contextClass)
+        $context = $this->getMockBuilder(ExecutionContext::class)
```
Comment (OskarStark) [inline tests/Validator/FormatValidatorTest.php]: please use `createMock()` whenever possible

Comment (jordisala1991) [reply tests/Validator/FormatValidatorTest.php]: Will try, but only on the code I changed, because this is not the objective of the PR

Comment (jordisala1991) [reply tests/Validator/FormatValidatorTest.php]: Done!

**Comment by greg0ire** [comment]
Thanks @jordisala1991 !

**Pull request merged**